### ISSUE TITLE
Task switcher stays classic after uninstall 

### ISSUE THREAD

**Issue by whudwl** [Open-Shell/Open-Shell-Menu#274]
**Describe the bug**
After uninstalling open shell, the windows 10 task switcher look is not restored to default. In other words, when I press alt + tab, I see only icons not previews and I couldn't click on these icons to switch tasks.  

**To Reproduce**
install open shell then uninstall. Check if the behavior and look of task switcher is restored. 

**Expected behavior**
all functions should be restored after uninstall.  

**Version:**
 - Open-Shell: [e.g. 4.4.131]
 - OS: [e.g. Windows 10 1903]

**Comment by ge0rdi** [comment]
Are you sure it is related to `Open-Shell`?
I'm not aware of any functionality that could affect _task switcher_ :(

Does it get fixed after reboot?

**Comment by whudwl** [comment]
I'm pretty sure. Windows does not offer a way to configure the task switcher, so I'd say it's highly likely it's open shell that changed the look and behavior of task switcher.   

To restore the look and behavior of task switcher, I had to change the value of AltTabSettings from 1 to 0 under HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer 

![20191216_182548](https://user-images.githubusercontent.com/2842789/70887413-01ddfb00-2032-11ea-9af6-9ddc092e7b04.png)

**Comment by ge0rdi** [comment]
> To restore the look and behavior of task switcher, I had to change the value of AltTabSettings from 1 to 0 under HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer

I can't find any reference of `AltTabSettings` in source code :(

> This is what task switcher looks like after installing open shell & uninstalling open shell.

So you can replicate this?
Try to start with _good_ state, install latest `Open-Shell` (check if the issue is present), then uninstall it and check again.

**Comment by whudwl** [comment]
I tried to install & uninstall open shell one more time, the problem didn't happen this time... 
Then I tried install & uninstall classic shell one more time, the problem didn't happen either... 
Sorry about this. It's probably some other software that messed with the task switcher.  I just assumed it was open shell cause it changes start menu to classic so I thought maybe it did the same to task switcher too.
